### PR TITLE
tilt: eth speedup

### PR DESCRIPTION
**Pull request by panoel**
This addresses issue #1825

What should be done with the "old" scripts?  Delete them or put them into an "old scripts" folder?

### CODE REVIEW

File: devnet/eth-devnet.yaml [hunk 1]
```diff
@@ -54,7 +54,8 @@ spec:
...
+            - "npm run migrate && npx truffle exec scripts/deploy_test_token.js && npm run deploy-batched-vaa-sender && npx truffle exec scripts/register_all_chains.js && nc -lkp 2000 0.0.0.0"
+            # - "npm run migrate && npx truffle exec scripts/deploy_test_token.js && npm run deploy-batched-vaa-sender && npx truffle exec scripts/register_solana_chain.js && npx truffle exec scripts/register_terra_chain.js && npx truffle exec scripts/register_terra2_chain.js && npx truffle exec scripts/register_bsc_chain.js && npx truffle exec scripts/register_algo_chain.js && npx truffle exec scripts/register_near_chain.js && npx truffle exec scripts/register_worm_chain.js && npx truffle exec scripts/register_aptos_chain.js && nc -lkp 2000 0.0.0.0"
```
Comment (jumpsiegel) [inline devnet/eth-devnet.yaml:58]: i guess remove this line?

Comment (panoel) [reply devnet/eth-devnet.yaml]: Removed

**Review by jumpsiegel — DISMISSED** [review]

**Review by jumpsiegel — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]
Let's read the env vars in a loop in the next PR 🙂

**Review by kcsongor — APPROVED** [review]

**Comment by Stenzzy** [comment]
Add

**Pull request merged**